tests/test_inter_area_agent.py

```diff
@@ -368,8 +366,8 @@ def iaa_double_sided():
     owner = FakeArea('owner')
     iaa = TwoSidedPayAsBidAgent(owner=owner, lower_market=lower_market,
                                 higher_market=higher_market)
-    iaa.engines[0]._match_offers_bids = lambda: None
-    iaa.engines[1]._match_offers_bids = lambda: None
+    iaa.engines[0].match_offers_bids = lambda: None
+    iaa.engines[1].match_offers_bids = lambda: None
```

Comment:
Since the matching is performed in market and triggered from area, I guess this check is not necessary inside inter_area_engine and above two lines could be omitted.

Reply:
Yes you are right, removing it.